src/odemis/acq/test/move_test.py

```diff
@@ -617,6 +618,151 @@ def test_moving_in_grid1_fm_imaging_area_after_loading(self):
                                                                    "the FM imaging grid 1 area.")
 
 
+class TestMeteorTFS2Move(TestMeteorTFS1Move):
+    """
+    Test the MeteorPostureManager functions for TFS 2
+    """
+    MIC_CONFIG = METEOR_TFS2_CONFIG
+    ROTATION_AXES = {'rx', 'rz'}
+
+    def test_moving_in_grid1_fm_imaging_area_after_loading(self):
+        """Check if the stage moves in the right direction when moving in the fm imaging grid 1 area."""
+        super().test_moving_in_grid1_fm_imaging_area_after_loading()
+        meteor_stage_md = self.linked_stage.getMetadata()
+        fav_pos_active_orig = meteor_stage_md.get(model.MD_FAV_POS_ACTIVE, None)
+        # check z in fm when moving in FM mode
+        self.assertAlmostEqual(self.linked_stage.position.value["z"], fav_pos_active_orig["z"], places=5)
+
+        # move in the same imaging mode using linked YM stage
+        old_stage_pos = self.stage.position.value
+        self.linked_stage.moveRel({"y": 1e-3}).result()
+        new_stage_pos = self.stage.position.value
+        # check z in fm when moving in FM mode
+        self.assertAlmostEqual(self.linked_stage.position.value["z"], fav_pos_active_orig["z"], places=5)
+
+        # the stage moved in the right direction if the pre-tilt angle was maintained at 26-degrees
+        beta = 0.8034946943806713  # 45-degrees in radians
+
+        estimated_beta = math.atan2(new_stage_pos["z"] - old_stage_pos["z"], new_stage_pos["y"] - old_stage_pos["y"])
+        self.assertAlmostEqual(beta, estimated_beta, places=5, msg="The stage moved in the wrong direction in "
+                                                                   "the FM imaging grid 1 area.")
+
+    def test_moving_from_grid1_to_grid2_in_fm_imaging_Area(self):
+
+        # todo check zfm in hardware
+        #-0.0017951474872928566 , -2.4472740404152816e-05, grid 2 zfm 0.0017706747468887037
+        meteor_stage_md = self.linked_stage.getMetadata()
+        self.linked_stage.updateMetadata({model.MD_FAV_POS_ACTIVE: {'z': 0.0017706747468887037}})
+        fav_pos_active_orig = self.linked_stage.getMetadata().get(model.MD_FAV_POS_ACTIVE, None)
+
+        f = self.posture_manager.cryoSwitchSamplePosition(LOADING)
```

Comment:
You should move to the SEM imaging first

Reply:
Is it not safe to move to FM from Loading position? Is there another reason we should not switch to FM directly from Loading? From this code it looks safe 
![image](https://github.com/delmic/odemis/assets/37955981/f94bf4a9-d822-4b2e-8f06-7e59b78d3d13)